packages/mobile/src/home/HomeTokenBalance.tsx

```diff
@@ -0,0 +1,147 @@
+import Colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Dialog from 'src/components/Dialog'
+import { calculateDecimalsToShow } from 'src/components/TokenDisplay'
+import { Namespaces } from 'src/i18n'
+import InfoIcon from 'src/icons/InfoIcon'
+import ProgressArrow from 'src/icons/ProgressArrow'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { tokensWithBalanceSelector, totalTokenBalanceSelector } from 'src/tokens/selectors'
+
+function TokenBalance() {
+  const tokenBalances = useSelector(tokensWithBalanceSelector)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const totalBalance = useSelector(totalTokenBalanceSelector)
+  if (tokenBalances.length === 0) {
+    return (
+      <Text style={styles.balance} testID={'TotalTokenBalance'}>
+        {localCurrencySymbol}
+        {'0.00'}
```

Comment:
The period depends on the locale, in some places (like Argentina) they use a comma instead. How about using `new BigNumber(0).toFixed(2)` instead?

Reply:
great suggestion!